stubs/python-requests/run.py

```diff
@@ -0,0 +1,15 @@
+import sys
+import requests
+
+
+host = sys.argv[1]
+port = sys.argv[2]
+cafile = sys.argv[3] if len(sys.argv) > 3 else False
+
+try:
+    r = requests.get("https://" + host + ":" + port, cert=cafile)
```

Comment:
If I'm looking the correct [docs](http://docs.python-requests.org/en/master/user/advanced/) for `requests`, the `cert` parameter is used for TLS's client certificate authentication. CA bundle is given with [`verify`](http://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification) parameter.